versioned_docs/version-v0.17.0/int/quickstart/monitoring.md

```diff
@@ -0,0 +1,94 @@
+# Getting Started Monitoring your Node
```

Comment:
Was this an accidental commit of something stashed, or was it meant to come in in this? if it was, why is it in the v0.17.0 folder? Generally stuff goes in the `docs` folder, and when a version is cut, it gets copied to the `versioned_docs` folder.